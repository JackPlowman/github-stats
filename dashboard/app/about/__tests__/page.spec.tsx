import { expect, test, jest } from "@jest/globals";
import { render } from "@testing-library/react";

import About from "../page";

test("renders about page", async () => {
  // Act
  const { findByText } = render(<About />);
  // Assert
  const heading = await findByText(/This is the about page/i);
  expect(heading).toBeTruthy();
  // check content
  expect(findByText).toMatchSnapshot();
});
