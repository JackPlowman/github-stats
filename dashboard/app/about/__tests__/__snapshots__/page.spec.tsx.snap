// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about page 1`] = `[Function]`;
